.gitignore

```diff
@@ -0,0 +1,8 @@
+go.mod
```

Comment:
This and `go.sum` should not be ignored.

Reply:
>`go.mod`
>A module is defined by a tree of Go source files with a go.mod file in the tree's root directory. 

>`go.sum`
>The go.mod files in a build provide enough information for 100% reproducible builds.

I see, thanks!